tools/ovc/openvino/tools/ovc/get_ov_update_message.py

```diff
@@ -16,33 +16,10 @@ def get_ov_update_message():
     return msg_fmt.format(link) if current_date >= expected_update_date else None
 
 
-def get_ov_api20_message():
-    link = "https://docs.openvino.ai/2023.0/openvino_2_0_transition_guide.html"
-    message = '[ INFO ] The model was converted to IR v11, the latest model format that corresponds to the source DL framework ' \
-              'input/output format. While IR v11 is backwards compatible with OpenVINO Inference Engine API v1.0, ' \
-              'please use API v2.0 (as of 2022.1) to take advantage of the latest improvements in IR v11.\n' \
-              'Find more information about API v2.0 and IR v11 at {}'.format(link)
-
-    return message
-
-
-def get_tf_fe_message():
-    link = "https://docs.openvino.ai/2023.0/openvino_docs_MO_DG_TensorFlow_Frontend.html"
-    message = '[ INFO ] IR generated by new TensorFlow Frontend is compatible only with API v2.0. Please make sure to use API v2.0.\n' \
-              'Find more information about new TensorFlow Frontend at {}'.format(link)
-
-    return message
```

Comment:
@popovaan, why this is removed? This limitation is still valid because we really not compatible with API 1.0 if using TF FE.

Reply:
New tool is introduced in the moment of time where is no API1.0 already so no need to mention old stuff

Reply:
API 1.0 still exists and released